CHANGELOG.md

```diff
@@ -34,7 +34,7 @@ release.
   ([#3402](https://github.com/open-telemetry/opentelemetry-specification/pull/3402))
     BREAKING: rename `net.peer.name` to `server.address` on client side and to `client.address` on server side,
      `net.peer.port` to `server.port` on client side and to `client.port` on server side,
-     `net.host.name` and `net.host.port` to `server.name` and `server.port` (since `net.host.*` attributes only applied to server instrumentation)
+     `net.host.name` and `net.host.port` to `server.address` and `server.port` (since `net.host.*` attributes only applied to server instrumentation),
```

Comment:
```suggestion
     `net.host.name` and `net.host.port` to `server.address` and `server.port` (since `net.host.*` attributes primarily applied to server instrumentation),
```
How about this then?

Reply:
This is IMHO not enough of a reason and should be changed to the more wordy:

```suggestion
     `net.host.name` and `net.host.port` to `server.address` and `server.port` on server side (typical case) and `client.address` and `client.port` on client side,
```

But that would need to be changed in all the other points below as well.

It seems odd that a CHANGELOG entry produces so much food for discussion, how about removing the mapping from here entirely and instead doing what #43 suggests with a proper document?

Reply:
Resolving for now per https://github.com/open-telemetry/semantic-conventions/pull/126#issuecomment-1602104246.

Reply:
(unresolving after merge for easier future reference)